edb/schema/delta.py

```diff
@@ -1470,6 +1471,53 @@ def _propagate_if_expr_refs(
 
         return schema
 
+    def _prohibit_if_expr_refs(
```

Comment:
I added a comment elsewhere indicating that I'm not 100% sure why we need to prohibit instead of propagate, but assuming we do, I think that `_propagate_if_expr_refs` could be made to prohibit by having a flag that disables the main `if isinstance(` branch?

If you prefer to have it separated out, the `fixer` argument is dead and can be dropped.